dev-system/dev/user.clj

```diff
@@ -251,7 +251,7 @@
 (defn stop
   "Shuts down and destroys the current development system."
   []
-  (alter-var-root #'system
+  (alter-var-root #'user/system
```

Comment:
Good catch.

Reply:
Not sure why we have to make this change.

Reply:
There's a require using the alias `system`, so that var was ambiguous. The problem I saw that was fixed by changing this is after calling user/stop directly I could not perform a refresh because my system was not actually stopped. After this change, calling stop did successfully stop my system.